app/templates/docs/card.html

```diff
@@ -1798,6 +1798,7 @@
   </h2>
   <ul class="simple">
     <p>The attachments endpoint should be used to add an attachment to a card. Cards are limited to having 100 attachments. You can only add a single attachment to a card at a time.</p>
+    <p>There are two primary ways of adding attachments to cards. The first is by attaching a URL directly to a card. To do so, you include the URL as a query parameter to the POST request. The second way requires that you pass in a file object in the POST request. Trello expects that the file object is encoded as multipart/form-data. For examples demonstating how to add attachments check out this <a href="https://glitch.com/~trello-attachments-api">Glitch project</a>!</p>
```

Comment:
I might trade the `!` at the end for something like 🚀  but that's a personal choice 😛